test/e2e/setup.go

```diff
@@ -118,6 +122,12 @@ func skipIfNotHiveManagedCluster(adminAPICluster *admin.OpenShiftCluster) {
 	}
 }
 
+func skipUnlessSmokeTestEnabled() {
+	if os.Getenv("ARO_E2E_SMOKE_TEST") == "" {
+		Skip("skipping smoke tests")
+	}
+}
+
```

Comment:
Rather than skipping these tests based on a env var what if we just ran the e2e tests with a `--label-filter="!smoke"`. Equally, we could label all the tests as `E2E` and run with `--label-filter="E2E"` but that seems like overkill.

What do you think?

Reply:
nice, it would be simpler.

https://onsi.github.io/ginkgo/#spec-labels